src/status_im2/navigation/roots.cljs

```diff
@@ -76,7 +76,7 @@
    {:root {:stack {:id       :onboarding
                    :children [{:component {:name    :get-your-keys
                                            :id      :get-your-keys
-                                           :options (status-bar-options)}}]
+                                           :options {:topBar {:visible false}}}}]
```

Comment:
I recommend creating new root instead of modifying old one. New on-boarding screen has different flows, so they don't work well together

Reply:
@Parveshdhull looks like different place for comment ?

Reply:
Looks like we are using same screen/root `:get-your-keys`, and just [replacing](https://github.com/status-im/status-mobile/pull/15110/files#diff-3ac98dabe1eb577bf32a1239246ceee1c531d6ee8701ce713ac22608d38556c9) its content. That doesn't look good.

Reply:
@flexsurfer curious how should we handle it?
- We can't just use new root, because it is incomplete and will break the flow.
- We shouldn't replace screens in old root, because it had different flow. And we have to use hacks like this to make it work
![image](https://user-images.githubusercontent.com/17097240/219686737-89c12bb1-08cc-4fe5-9f49-c0e30b1dfd0d.png)

(Maybe we can create sandboxed on-boarding and replace with current one, ones it is complete) wdyt?
cc: @ulisesmac @J-Son89

Reply:
@Parveshdhull these questions not related to this code line `{:topBar {:visible false}}` right ?

Reply:
> [@Parveshdhull](https://github.com/Parveshdhull) these questions not related to this code line {:topBar {:visible false}} right ?

They are related.
Here inline `{:topBar {:visible false}` @ulisesmac removing topBar from old `:get-your-keys` screen and using the same screen for "new to status" screen. ([code](https://github.com/status-im/status-mobile/pull/15110/files#diff-3ac98dabe1eb577bf32a1239246ceee1c531d6ee8701ce713ac22608d38556c9))

Sorry, you missed the context because part of the conversation happened on discord (added you to the group). Please let me know if anything is not clear

Reply:
I personally would prefer if we can bridge the two flows (old and new) and slowly migrate away from the old. But happy to do this either way.

Reply:
@flexsurfer @J-Son89 @Parveshdhull 
To solve this, I'll wait for @J-Son89 first onboarding screen implementation, and continue the flow created.